includes/wf_crm_webform_postprocess.inc

```diff
@@ -966,6 +966,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           $perm = $perm == 1 ? 2 : 1;
         }
       }
+      //initialise start_date for create action.
+      if (empty($existing) !array_key_exists('start_date', $params) {
```

Comment:
I'm having trouble reading this if statement. There's no operator between the two conditions.
Also it looks like there's a missing closing bracket. Did you try running this code? Looks like you'd get a fatal syntax error.